src/components/Calendar/index.jsx

```diff
@@ -0,0 +1,77 @@
+import { useCallback, useEffect, useRef, useState } from "react";
+
+import NEXT_BUTTON from "@assets/nextButton.svg";
+import PREV_BUTTON from "@assets/prevButton.svg";
+import CalendarPage from "@components/Calendar/CalendarPage";
+import Icon from "@components/common/Icon";
+import { throttle } from "@utils/";
+import { getDirectionValue } from "@utils/calendar";
+
+import * as S from "./style";
+
+const Calendar = () => {
+  // pageIndex는 더 상위에서 관리해야 함. 초기화되면 안됨.
+  const [pageIndex, setPageIndex] = useState(0);
+  const [slideInfo, setSlideInfo] = useState({
+    translateX: -50 + pageIndex * 50,
+    direction: null,
+  });
+  const isMovePending = useRef(false);
+
+  // const handleMoveCalendar = useCallback(
+  //   throttle((movePoint, direction) => {
+  //     setSlideInfo(({ translateX }) => ({ translateX: translateX + movePoint, direction: direction }));
+  //     setPageIndex((prev) => prev + getDirectionValue(direction));
+  //   }, 300),
+  //   [slideInfo],
+  // );
+
+  const handleMoveCalendar = useCallback(
+    (movePoint, direction) => {
+      if (!isMovePending.current) {
+        isMovePending.current = true;
+        setSlideInfo(({ translateX }) => ({ translateX: translateX + movePoint, direction: direction }));
+        setPageIndex((prev) => prev + getDirectionValue(direction));
+      }
+    },
+    [isMovePending.current],
+  );
+
+  const handleMoveEnd = () => {
+    isMovePending.current = false;
+  };
+
+  console.log(pageIndex);
+
+  const today = new Date();
+  const todayYear = today.getFullYear();
+  const todayMonth = today.getMonth();
+  const todayDate = today.getDate();
+  const getTodayDatePassByMonth = useCallback((passMonth) => new Date(todayYear, todayMonth + passMonth), []);
```

Comment:
```suggestion
  const getTodayDatePassByMonth = useCallback((passMonth) => new Date(todayYear, todayMonth + passMonth), [
    todayYear,
    todayMonth,
  ]);
```

의존성 배열에 의존하는 값을 꼭 명시해주세요. eslint rule을 적용하시는 걸 추천드립니다.(ref: https://github.com/facebook/react/issues/14920)